winton_kafka_streams/processor/_stream_task.py

```diff
@@ -66,8 +67,14 @@ def __init__(self, _task_id, _application_id, _partitions, _topology_builder, _c
         self.commitOffsetNeeded = False
         self.consumedOffsets = {}
 
+        self._init_state_stores()
         self._init_topology(self.context)
 
+    def _init_state_stores(self):
+        self.log.debug(f'Initialising state stores')
+        for store in self.topology.state_stores.values():
+            store.initialise(self.context, store)
```

Comment:
The state stores get another initialise call now, similar to Processors. This is because they need the context, which isn't available at the time when the topology is being built.